docs/ref/contrib/admin/index.txt

```diff
@@ -2549,6 +2567,8 @@ so by defining an ``InlineModelAdmin`` object for the relationship::
 
     from django.contrib import admin
 
+    from .models import Group
```

Comment:
```suggestion
    from .models import Group, Person
```

Reply:
Actually for this example specifically, I think it's worth us only defining `GroupAdmin` to keep this focussed on what it's trying to show so:
```
    from django.contrib import admin

    from .models import Group


    class MembershipInline(admin.TabularInline):
        model = Group.members.through


    class GroupAdmin(admin.ModelAdmin):
        inlines = [
            MembershipInline,
        ]
        exclude = ["members"]


    admin.site.register(Group, GroupAdmin)
```

Reply:
True